lib/cfn-nag/custom_rules/SageMakerEndpointConfigKmsKeyIdRule.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'missing_property_base_rule'
+
+class SageMakerEndpointConfigKmsKeyIdRule < MissingPropertyBaseRule
+  def rule_text
+    'SageMaker EndpointConfig must have a KmsKeyId property set.'
```

Comment:
not sure we've ever been consistent about this, but "must" sounds a fail vs. a warning.  maybe "should" instead?

Reply:
Good point!  I'll update to "should" in the next commit.